tox.ini

```
@@ -1,9 +1,9 @@
 [tox]
 envlist =
-    py{37,38,39,310}-django32
+    py{38,39,310}-django32
+    py{38,39,310,311}-django{41,42}
```

this can be made a bit shorter:

```suggestion
    py{38,39,310}-django{32,41,42}
    py311-django{41,42}
```
when we drop 3.2 support, this will then become `py{38,39,310,311}-django{41,42}`